### PR TITLE
New package: Kubernetes.kubectl version 1.24.3

### LINKED ISSUE #30129: [Package Request]: Kubectl
### Package Requested

- [ ] I would like someone else to build the manifest.
- [X] I would like help so I can submit the manifest.
- [X] I have performed a search and couldn't find this package.
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: Kubernetes
Publisher Url: https://kubernetes.io
Package Name: Kubernetes.kubectl
Description: The Kubernetes command-line tool
Package Version: 1.22.0
Installer URL: https://dl.k8s.io/release/v1.22.0/bin/windows/amd64/kubectl.exe

Current Package Identifier (for new versions):

### PR DESCRIPTION
**Pull request by russellbanks**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

* Resolves #30129

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/69105)

### CODE REVIEW

File: manifests/k/Kubernetes/kubectl/1.24.3/Kubernetes.kubectl.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+License: Apache License 2.0
+LicenseUrl: https://github.com/kubernetes/kubectl/blob/master/LICENSE
+Copyright: Copyright (c) The Kubernetes Authors
+# CopyrightUrl: 
+ShortDescription: The Kubernetes command-line tool
+# Description: 
+Moniker: kubectl
+Tags:
+- kubectl
+- kubernetes
```
Comment (Trenly) [inline manifests/k/Kubernetes/kubectl/1.24.3/Kubernetes.kubectl.locale.en-US.yaml:23]:
```suggestion
# Tags:
```
Moniker and Package Identifier will take precedence over the tags, so these tags are redundant

File: manifests/k/Kubernetes/kubectl/1.24.3/Kubernetes.kubectl.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+PublisherSupportUrl: https://github.com/kubernetes/kubectl/issues
+# PrivacyUrl: 
+Author: The Kubernetes Authors
+PackageName: kubectl
+PackageUrl: https://kubernetes.io/docs/reference/kubectl/
+License: Apache License 2.0
+LicenseUrl: https://github.com/kubernetes/kubectl/blob/master/LICENSE
+Copyright: Copyright (c) The Kubernetes Authors
+# CopyrightUrl: 
+ShortDescription: The Kubernetes command-line tool
```
Comment (Trenly) [inline manifests/k/Kubernetes/kubectl/1.24.3/Kubernetes.kubectl.locale.en-US.yaml:18]:
```suggestion
ShortDescription: A command line tool for communicating with a Kubernetes cluster's control plane, using the Kubernetes API
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**